Common/Models/Comments/Comment.cs

```diff
@@ -9,5 +10,7 @@ public class Comment : IComment
         public string Author { get; set; }
 
         public string Message { get; set; }
+
+        public IProduct Product { get; set; }
```

Comment:
I am not sure about this one: do we really need to store the entire Product inside the Comment?
It seems to me that it would be better to store just a ProductId here instead.

`public int ProductId { get; set; }`

- [ ] Replace Product property with ProductId;
- [ ] Make changes all over the code to use just `int` value type for the reference, instead of bulky `IProduct' reference type.